runtime/plugin/lua.vim

```diff
@@ -0,0 +1 @@
+lua vim.inspect = require("vim.inspect")
```

Comment:
Let's name this file `runtime/plugin/nvim.vim` . We may want to do other non-Lua things in it.